functests/1_performance/performance.go

```diff
@@ -50,6 +50,49 @@ var _ = Describe("[rfe_id:27368][performance]", func() {
 		Expect(err).ToNot(HaveOccurred())
 	})
 
+	Context("Tuned CRs generated from profile", func() {
+		It("[test_id:] Should have the expected name for tuned from the profile owner object", func() {
+			tunedExpectedName := components.GetComponentName(testutils.PerformanceProfileName, components.ProfileNamePerformance)
+			tunedList := &tunedv1.TunedList{}
+			err := testclient.Client.List(context.TODO(), tunedList)
+			Expect(err).NotTo(HaveOccurred())
+
+			key := types.NamespacedName{
+				Name:      components.GetComponentName(testutils.PerformanceProfileName, components.ProfileNamePerformance),
+				Namespace: components.NamespaceNodeTuningOperator,
+			}
+			tuned := &tunedv1.Tuned{}
+			err = testclient.Client.Get(context.TODO(), key, tuned)
+			Expect(err).ToNot(HaveOccurred(), "cannot find the Cluster Node Tuning Operator object "+components.ProfileNamePerformance)
+			outdatedTuned := tuned.DeepCopy()
+			outdatedTuned.Name = "outdatedName"
+			err = testclient.Client.Create(context.TODO(), outdatedTuned)
+			Expect(err).NotTo(HaveOccurred())
+
+			Eventually(func() bool {
+				for t := range tunedList.Items {
```

Comment:
you need to list again under the Eventually, otherwise, you always will have old list

Reply:
I also afraid of the race here:
- tuned created
- the list still does not have it
- you call to get and it does not return the error, because the operator wasn't fast enough to delete it